cli.py

```diff
@@ -1,4 +1,27 @@
 import click
+import os
+from ml.base import MLConfig, MLEstimator
 
-def main():
-    pass
+_CONFIG_PATH = os.path.join(os.getcwd(), 'configs/')
+
+@click.command()
```

Comment:
I think the right way is 

@click.command()
def train(args):
    pass

then the function name will be directly used as the command name.

Reply:
The click decorator will take care of the CLI arguments and the actions can be handled by `click.argument` decorator. I'll write tests for it anyway.